recipe/meta.yaml

```
@@ -25,8 +25,8 @@ requirements:
   run:
     - aiohttp >=3.8.3,<4.0.0
     - async-timeout >=4.0.0,<5.0.0  # [py<311]
-    - langchain-core >=0.3.21,<0.4.0
-    - langchain-text-splitters >=0.3.0,<0.4.0
+    - langchain-core >=0.3.25,<0.4.0
+    - langchain-text-splitters >=0.3.3,<0.4.0
     - langsmith >=0.1.17,<0.2.0
```

https://github.com/langchain-ai/langchain/blob/langchain%3D%3D0.3.12/libs/langchain/pyproject.toml#L38

```suggestion
    - langsmith >=0.1.17,<0.3.0
```

---

Missed that one. Thanks!